solutions/standard/variables.tf

```diff
@@ -62,6 +62,18 @@ variable "secret_manager_tags" {
   default     = []
 }
 
+variable "iam_engine_enabled" {
+  type        = bool
+  description = "Set this to true to to configure an IBM Secrets Manager IAM credentials engine for an existing IBM Secrets Manager instance. If set to false, no iam engine will be configured for your secrets manager instance. For more details, see https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-configure-iam-engine."
```

Comment:
Maybe don't say `for an existing IBM Secrets Manager instance` - the instance is provisioned as part of the DA solution